cypress/tests/core/volto-slate/09-block-slate-slashmenu.js

```diff
@@ -0,0 +1,18 @@
+import { slateBeforeEach } from '../../../support/e2e';
+
+describe('SlashMenu Test: Shortcuts', () => {
+  beforeEach(slateBeforeEach);
+
+  it('As editor I can crate a Description block using the SlashMenu shortcut', function () {
+    // Use SlashMenu shorcut to create a Description block and type some text in it
+    cy.getSlateEditorAndType('/D').type('{enter}');
```

Comment:
I'd suggest adding more of the block title, just to make the test more robust in case other blocks with the letter D are added in the future and it's no longer the first option.

```suggestion
    cy.getSlateEditorAndType('/Desc').type('{enter}');
```